docs/deploy.rst

```diff
@@ -0,0 +1,217 @@
+
+This list shows your region's available CrateDB products. There are five tiers
+available for the deployment of CrateDB products: XS, S, M, L, XL.
+
+For this test deployment, pick tier ``xs`` which is the smallest available tier.
+According to the list above, this is the CrateDB cluster ``cratedb.az-gp1``.
+
+
+.. _launch-cluster:
+
+Launch the cluster
+==================
+
+Now that you've set up an organization and project, you can deploy your first
+cluster. Give this command:
+
+.. code-block:: console
+
+   sh$ croud clusters deploy \
+       --product-name cratedb.az-gp1 \
+       --tier xs \
+       --cluster-name my-first-crate-cluster \
+       --project-id 59c7961b-33a2-4478-9e98-2a6b3f54f60e \
+       --version 3.3.0 \
+       --username yourusername \
+       --password "yourpassword" \
+       --region bregenz.a1
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | id                                   | name                   | numNodes | crateVersion| projectID                            | username | fqdn                                                 |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+   | 8d6a7c3c-61d5-11e9-a639-34e12d2331a1 | my-first-crate-cluster |        1 | 3.3.0       | 59c7961b-33a2-4478-9e98-2a6b3f54f60e | default  | my-first-crate-cluster.bregenz.a1.cratedb.net. |
+   +--------------------------------------+------------------------+----------+-------------+--------------------------------------+----------+------------------------------------------------------+
+  ==> Success: Cluster deployed. It may take a few minutes to complete the
+  changes.
+
+The cluster deployment will take around 5-10 minutes to complete.
+
+.. TIP::
+
+  Look up the `Croud CLI Reference`_ to learn what each argument in the
+  ``clusters`` command specifies.
+
+If the operation was successful you'll get the deployment details returned.
+
+Once the cluster is successfully deployed, you'll be asked to log in.
+
+Enter your CrateDB Cloud login credentials and select *Sign in*.
+
+The CrateDB administration user interface, admin UI for short, opens and
+shows the CrateDB cluster that you've deployed. You can use it to inspect and
+interact with the whole CrateDB cluster in a number of ways.
+
+Look up the corresponding section to learn everything surrounding the
+`CrateDB admin UI`_.
+
+.. image:: _assets/img/admin-ui.png
+
+.. _what-was-learned:
+
+Next steps
+==========
+
+Congratulations! You've deployed your first CrateDB cluster. Why don't you take
+a closer look at the `Cloud Console`_ web administration interface and learn how
```

Comment:
this might be a nice place to show a screenshot of the console if you do log in to whet their appetite for following this suggestion

like ooh look new shiny stuff to play with